### ISSUE TITLE
test: Missing argument at index 2

### ISSUE THREAD

**Issue by Pitasi** [franciscolourenco/done#86]
Hi, since this morning I started to have errors at the end of every command output:

```
$  sleep 6; echo yay
yay
test: Missing argument at index 2

~/.config/fish/conf.d/done.fish (line 180):
                if test -q __done_notification_urgency_level
                   ^
in function '__done_ended'
in event handler: handler for generic event “fish_prompt”

(Type 'help test' for related documentation)
```

not sure what caused the issue. Alerts are working fine as usual (both normal and errors).

I'm using tmux 3.1b, fish 3.1.2, done 1.14.7.

**Comment by ammgws** [comment]
I think that's meant to be `set -q` not `test -q`

**Comment by franciscolourenco** [comment]
Thank you for catching this, and sorry for releasing with this issue. Fixed in v1.14.8

**Comment by Pitasi** [comment]
No worries, thanks to you for such a fast fix :smile:
